docs/reference/install-kubernetes.md

```diff
@@ -51,8 +51,8 @@ $ kubectl gadget version
 ```
 
 ## Installing in the cluster
-Before deploying make sure that kubernates cluster is deployed . Like minikube is running .\
-Steps to make minikube running - [minikube](https://minikube.sigs.k8s.io/docs/start/?arch=%2Fwindows%2Fx86-64%2Fstable%2F.exe+download)
+
+Before you begin, ensure that your Kubernetes cluster is up and running and that you can access it with `kubectl`.
```

Comment:
done

Reply:
Mmmmm he intended to squash everything in one single commit 🙂.

Reply:
I made the latest commit  which contain the required change.

Reply:
should I create a new branch and then commit one message on it? Because on this branch in the commit history [o: Bump github.com/containerd/containerd from 1.7.20 to 1.7.21](https://github.com/inspektor-gadget/inspektor-gadget/pull/3412/commits/c52dc0a2ec86e38983281645954f19cf84beb797) is also present?

Reply:
![Screenshot 2024-08-29 150843](https://github.com/user-attachments/assets/4a0af983-3752-4f8c-9dae-e83975506760)
![Screenshot 2024-08-29 150853](https://github.com/user-attachments/assets/a300a31b-8cbb-41d4-8178-817df68b9d8e)
When i followed commands ,when the editor opened it contain 79 commits . So i should replace my commits with drop remaining only one commit? Rest commits which are not pushed my me i leave them as it is.

Reply:
you need to resolve your current index first. 
 I am not understanding why it is happening . For the time being should i raise PR from other branch and later on figure out what problem is happening here?

Reply:
https://github.com/inspektor-gadget/inspektor-gadget/pull/3424 I have made new PR.

Reply:
https://github.com/inspektor-gadget/inspektor-gadget/pull/3425 . Commited one message in new PR.